services/app/src/pages/game/Game.jsx

```diff
@@ -103,7 +99,10 @@ function Room() {
         </Team>
         <ChessBoard />
       </div>
-      <GameFooter />
+      <GameFooter 
+        skipMove={skipMove}
```

Comment:
Instead of passing a function (`skipMove`) and a parameter to that function (`currentPlayerId`) you could just pass a single function that combines the two:
```
const skipCurrentPlayerMove = () => skipMove(currentPlayerId);
...
<GameFooter skipMove={skipCurrentPlayerMove} />
```